graylog2-server/src/main/java/org/graylog2/Configuration.java

```diff
@@ -158,6 +159,11 @@
     @Parameter(value = "deactivated_builtin_authentication_providers", converter = StringSetConverter.class)
     private Set<String> deactivatedBuiltinAuthenticationProviders = Collections.emptySet();
 
+    // Defaults to TLS protocols that are currently considered secure
+    @Parameter(value = "enabled_tls_protocols", converter = StringSetConverter.class)
```

Comment:
We should add an entry to `misc/graylog.conf` including a comment that describes the available options.